core/services/vrf/delegate.go

```diff
@@ -185,6 +189,16 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 	return nil, errors.New("invalid job spec expected a vrf task")
 }
 
+// CheckFromAddressesExist returns an error if and only if one of the addresses
+// in the VRF spec's fromAddresses field does not exist in the keystore.
+func CheckFromAddressesExist(jb job.Job, gethks keystore.Eth) (err error) {
+	for _, a := range jb.VRFSpec.FromAddresses {
+		_, err2 := gethks.Get(a.Hex())
+		err = multierr.Append(err, err2)
```

Comment:
Go 1.20? Also, do we know if the address is included in this error, or do we want to wrap the error returned to include it? Otherwise it will return multiple non-descriptive errors.

Reply:
I believe the address is included here: https://github.com/smartcontractkit/chainlink/blob/7c68230773d4771a103702d480dff036e465b081/core/services/keystore/eth.go#L553 but I can include it again.

Reply:
Nope LGTM